MAUI/Popup/localization.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: Localization in .NET MAUI SfPopup control | Syncfusion
+description: Learn here all about Localization support in Syncfusion .NET MAUI Popup (SfPopup) control and more.
```

Comment:
Description: Learn about Localization support in Syncfusion .NET MAUI Popup (SfPopup) control and more.

Reply:
modified